src/main/java/teammates/storage/entity/FeedbackSession.java

```diff
@@ -138,6 +152,19 @@ public FeedbackSession(String feedbackSessionName, String courseId,
         this.respondingStudentList = studentList == null ? new HashSet<String>() : studentList;
     }
 
+    @OnLoad
+    @SuppressWarnings("unused") // called by Objectify
+    private void populateUtcFieldsFromLegacyFieldsIfRequired() {
```

Comment:
Objectify can invoke private methods? I'm quite surprised.

Reply:
Yup; it [uses](https://github.com/objectify/objectify/blob/4b6f4b9c1a7405be027602d556fda1a172e776a7/src/main/java/com/googlecode/objectify/impl/translate/LifecycleMethod.java#L19) reflection's `setAccessible` method.